### DISCUSSION TITLE
mantine version 7.0 module error

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [remix-run/remix#7486] [Q&A]
Since remix framework switched to esm in version 2 and mantine 7.0 supports esm then do you know why am i getting this error? 

    ` import { Button } from "@mantine/core";
                      ^^^^^^^^^^
        SyntaxError: Named export 'Button' not found. The requested module '@mantine/core' is a CommonJS module, which may not support all module.exports as named exports.
        CommonJS modules can always be imported via the default export, for example using:

        import pkg from '@mantine/core';
        const { Button } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:122:21)
    at ModuleJob.run (node:internal/modules/esm/module_job:188:5)
    at DefaultModuleLoader.import (node:internal/modules/esm/loader:228:24)
    at importModuleDynamicallyWrapper (node:internal/vm/module:428:15)
`

**Comment by VerseVent** [comment]
I just remove 7.0 package and used `yarn add @mantine/dropzone@6.0.21`. As a temporary solution, but if you want especially 7.0 idk ;)))
